modules/text/src/erfilter.cpp

```diff
@@ -47,6 +47,9 @@
 #include <fstream>
 #include <queue>
 
+#undef min
+#undef max
```

Comment:
It doesn't look reliable, especially there are no tesseract headers on the lines above.
Other files could be broken in the similar way easily by using normal `std::min()` / `std::max()`.

Please try this patch instead: https://github.com/alalek/opencv_contrib/commit/issue_2084

Reply:
Your patch resolves this bug. Build was successful.